packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/ToggleSwitch.tsx

```diff
@@ -96,15 +104,20 @@ export const ToggleSwitch = ({
 	};
 
 	return (
-		<label id={labelId} css={[labelStyles, cssOverrides]} {...props}>
+		<label
+			id={labelId}
+			css={[labelStyles(disabled), cssOverrides]}
```

Comment:
We prefer to [avoid dynamic styling](https://guardian.github.io/source/?path=/story/contributing-overview--page#using-emotion) where possible and favour composition of styles, e.g.

```tsx
// ToggleSwitch.tsx

<label
	id={labelId}
	css={[
		labelStyles, 
		disabled ? disabledLabelSyles : '',
		cssOverrides
	]}
>

// styles.ts

export const disabledLabelStyles = css`
	color: neutral[86];
`
```